build/NetStandardRelease.targets

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <NoWarn>RS0100</NoWarn>
```

Comment:
If you also add RS0016 here, then we can completely remove the PublicAPI files. The only API that we officially support is the Automation API, so that feels like the only place where the PublicAPI files seem to make sense.

Reply:
Done. I didn't add the files for the automation API though. If we want to adopt that practice in the future, we can have a conversation.